hack/lib/logging.sh

```diff
@@ -20,7 +20,7 @@ KUBE_VERBOSE="${KUBE_VERBOSE:-5}"
 # Handler for when we exit automatically on an error.
 # Borrowed from https://gist.github.com/ahendrix/7030300
 kube::log::errexit() {
-  local err="${PIPESTATUS[@]}"
+  local err="${PIPESTATUS[*]}"
```

Comment:
Reasons for modification:
```
In ./hack/lib/logging.sh line 23:
  local err="${PIPESTATUS[@]}"
            ^-- SC2124: Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.
```